src/OidcClient.ts

```diff
@@ -165,6 +165,18 @@ export class OidcClient {
         return response;
     }
 
+    public async processResourceOwnerPasswordCredentials(
```

Comment:
would be nice to have the params packed into one object:
```
/**
 * @public
 */
export type ProcessResourceOwnerPasswordCredentialsArgs = {
    username: string;
    password: string;
    skipUserInfo?: boolean;
};
...
    public async processResourceOwnerPasswordCredentials({
        username: string,
        password: string,
        skipUserInfo: boolean
    }: ProcessResourceOwnerPasswordCredentialsArgs): Promise<SigninResponse> {
````

Reply:
Fixed.